src/validation.cpp

```diff
@@ -757,6 +757,11 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
 
         //////////////////////////////////////////////////////////// // qtum
         if(tx.HasCreateOrCall()){
+
+            if(!CheckSenderScript(view, tx)){
+                return state.DoS(100, false, REJECT_INVALID, "bad-txns-invalid-sender-script");
```

Comment:
Make this a state.Invalid error, rather than DoS. We shouldn't DoS ban people who weren't aware of this restriction